### PR TITLE
➕ Add stylelint

### PR DESCRIPTION
**Pull request by cboenke**
https://stylelint.io/user-guide/get-started

- Install stylelint using: `npm install --save-dev stylelint stylelint-config-standard`
- Create `.stylelintrc.json`; for content see user-guide
- Add script for stylelint: `"stylelint": "stylelint \"**/*.css\""`
- Add [prettier config](https://github.com/prettier/stylelint-config-prettier)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check",
```
Comment (phlgr) [inline package.json]: Not sure how useful this script is.

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by m-wiemers — APPROVED** [review]
that`s pretty nice

File: package.json (hunk 1 above)
Comment (m-wiemers) [reply package.json]: #metoo

**Review by mrana17 — APPROVED** [review]
great 🚀

**Review by christophvogel — APPROVED** [review]
GJ 👍

**Comment by DNaunin** [comment]
LGTM! 🚀

**Pull request merged**